### PR TITLE
November news

### PR DESCRIPTION
**Pull request by maelle**
@steffilazerte Beside the usual human-generated content I added a new sentence at the bottom, about donations.

### CODE REVIEW

File: content/blog/2023-11-24-news-november-2023/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,647 @@
...
+Hosted by Steffi LaZerte and various community hosts. 
+Everyone welcome. 
+No RSVP needed. 
+Consult our [Events](/events) page to find your local time and how to join.
+
+- [Tuesday, December 5th, 9:00 Australian Western (01:00 UTC)](/events/coworking-2023-12/), Why won't it pass?! Troubleshooting R package checks. With community host [Adam Sparks](/author/adam-sparks/) and [Steffi LaZerte](/author/steffi-lazerte/).
+    - Explore what R package checks are and where they might fail;
+    - Work on an R package and run some checks;
+    - Spend some time troubleshooting your own failing package checks.
+
```
Comment (steffilazerte) [inline content/blog/2023-11-24-news-november-2023/index.Rmd:65]:
```suggestion

- [Tuesday, January 9th, 14:00 Europe Central (13:00 UTC)](/events/coworking-2024-01/), Working with APIs. With community host [Jon Harmon](/author/jon-harmon/) and [Steffi LaZerte](/author/steffi-lazerte/).
    - Explore using APIs in your own work;
    - Learn about packages that can help;
    - Discuss using APIs in R with our community host and other attendees.
    
```

File: content/blog/2023-11-24-news-november-2023/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,647 @@
...
+Join us for social coworking & office hours monthly on first Tuesdays! 
+Hosted by Steffi LaZerte and various community hosts. 
+Everyone welcome. 
+No RSVP needed. 
+Consult our [Events](/events) page to find your local time and how to join.
+
+- [Tuesday, December 5th, 9:00 Australian Western (01:00 UTC)](/events/coworking-2023-12/), Why won't it pass?! Troubleshooting R package checks. With community host [Adam Sparks](/author/adam-sparks/) and [Steffi LaZerte](/author/steffi-lazerte/).
+    - Explore what R package checks are and where they might fail;
+    - Work on an R package and run some checks;
+    - Spend some time troubleshooting your own failing package checks.
```
Comment (steffilazerte) [inline content/blog/2023-11-24-news-november-2023/index.Rmd:64]:
```suggestion
    - Spend some time troubleshooting your own failing package checks;
    - Discuss tips and tricks for troubleshooting package checks with our community host and other attendees.
```

File: content/blog/2023-11-24-news-november-2023/index.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+     * [naijR](https://github.com/ropensci/software-review/issues/600), Operations to Ease Data Analyses Specific to Nigeria. Submitted by [Victor Ordu ](https://victorordu.wordpress.com). 
+
+    * [octolog](https://github.com/ropensci/software-review/issues/502), Better Github Action Logging. Submitted by [Jacob Wujciak-Jens](https://github.com/assignUser). 
+
+* Eight at ['3/reviewer(s)-assigned'](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A3/reviewer(s)-assigned):
+
+     * [comtradr](https://github.com/ropensci/software-review/issues/613), Interface with the United Nations Comtrade API. Submitted by [paulbochtler](https://github.com/datapumpernickel). 
+
+    * [GLMMcosinor](https://github.com/ropensci/software-review/issues/603), Fit a cosinor model using a generalised mixed modelling framework. Submitted by [Rex Parsons](https://rwparsons.github.io/).
```
Comment (steffilazerte) [inline content/blog/2023-11-24-news-november-2023/index.md:89]: Duplicated below in this bullet

**Review by steffilazerte — COMMENTED** [review]
Looks great @maelle, I absolutely loved the dev corner (I learned so much!).

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary

File: content/blog/2023-11-24-news-november-2023/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,647 @@
...
+
+### Giving season
+
+PLACEHOLDER FOR SUMMARY OF YANI'S POST
+
+### Web assembly and R-universe
+
+PLACEHOLDER, JEROEN TO WRITE A TECH NOTE
+
+### Coworking
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2023-11-24-news-november-2023/index.Rmd:51]: please add it!

File: content/blog/2023-11-24-news-november-2023/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,647 @@
...
+Dear rOpenSci friends, it's time for our monthly news roundup!
+<!-- blabla -->
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### Giving season
+
+PLACEHOLDER FOR SUMMARY OF YANI'S POST
```
Comment (yabellini) [inline content/blog/2023-11-24-news-november-2023/index.Rmd:45]:
```suggestion

This Giving Season consider donating to rOpenSci to support our mission of empowering Open Science.

Sustaining an open project with quality infrastructure freely accessible to the global and diverse community of R software users, research software developers, and engineers requires many different resources. Our organization’s ongoing costs are supported by grants and donations from individuals and organizations which share our vision and mission.

By supporting us, you’re not just donating; you’re contributing to a community that’s breaking down barriers, creating opportunities, and shaping the future of open and reproducible science for everyone. 

Join us in this vital mission today! You can donate here: https://ropensci.org/donate/
```

File: content/blog/2023-11-24-news-november-2023/index.Rmd [hunk 6]
```diff
@@ -0,0 +1,647 @@
...
+
+- **[rvertnet](https://cran.r-project.org/web/packages/rvertnet/index.html)**, Retrieve, map and summarize data from the VertNet.org archives (<https://vertnet.org/>). Functions allow searching by many parameters, including taxonomic names, places, and dates. In addition, there is an interface for conducting spatially delimited searches, and another for requesting large datasets via email. [Issue for volunteering](https://github.com/ropensci-archive/rvertnet/issues/71).
+
+### Call for co-maintainers
+
+Refer to our somewhat [recent blog post](/blog/2022/10/17/maintain-or-co-maintain-an-ropensci-package/#packages-looking-for-co-maintainers) to identify other packages where help is especially wished for!
+See also our [help wanted page](/help-wanted/) -- before opening a PR, we recommend asking in the issue whether help is still needed.
+
+## Package development corner
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2023-11-24-news-november-2023/index.Rmd:605]: :smile_cat: thank you!

**Pull request merged**